checkstyle-checks/pom.xml

```diff
@@ -0,0 +1,52 @@
+<!--
+
+    Copyright (c) 2011-present Sonatype, Inc. All rights reserved.
+    Includes the third-party code listed at http://links.sonatype.com/products/clm/attributions.
+    "Sonatype" is a trademark of Sonatype, Inc.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.sonatype.buildsupport</groupId>
+    <artifactId>public-parent</artifactId>
+    <version>12</version>
+    <relativePath />
+  </parent>
+
+  <groupId>com.sonatype</groupId>
+  <artifactId>checkstyle-checks</artifactId>
+  <version>1-SNAPSHOT</version>
+
+  <name>Checkstyle Checks</name>
+  <description>
+    A checkstyle configuration that supports Sonatype's code style.
+  </description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
```

Comment:
Why not 8?

Reply:
> Why not 8?

We still have at least one project that is on Java 7 (https://github.com/sonatype/insight-scanner-hashing/blob/master/pom.xml) so wanted to be able to run it on all projects.

Reply:
If this were set to 1.8 I would expect it to mean the jar will be built for Java 8, requiring maven run in Java 8.

In other words, you could still build a Java 7 project with maven and a Java 8 version of this plugin, you'd just need to run maven itself in a Java 8 JVM. 

And if only one project is Java 7 it seems reasonable Java 8's already engineer's machines.

Reply:
> In other words, you could still build a Java 7 project with a Java 8 version of this plugin, you'd just need to run maven itself in a Java 8 JVM.

Yeah, which we do in Jenkins/Bamboo too.  Updated to 1.8 in 2536e190aa19981f8c80b8e58ae32e6d505d5ea9